health-services/project-factory/src/server/templateClasses/user-templateClass.ts

```
@@ -0,0 +1,163 @@
+
+// This will be a dynamic template class for different types
+export class TemplateClass {
+    // Static generate function
+    static async generate(templateConfig: any, responseToSend: any, localizationMap: any): Promise<SheetMap> {
+        logger.info("Generating template...");
+        logger.info(`Response to send ${JSON.stringify(responseToSend)}`);
+        const campaignDetailsResponse: any = await searchProjectTypeCampaignService({ tenantId: responseToSend.tenantId, ids: [responseToSend?.campaignId] });
+        const campaignDetails: any = campaignDetailsResponse?.CampaignDetails?.[0];
+        const readMeConfig = await getReadMeConfig(responseToSend.tenantId, responseToSend.type);
+        const readMeColumnHeader = getLocalizedName(Object.keys(templateConfig?.sheets?.[0]?.schema?.properties)?.[0], localizationMap);
+        const readMeData: any = this.getReadMeData(readMeConfig, readMeColumnHeader, localizationMap);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)